tests/deltatuner/cicd/run_ssf_model.sh

```diff
@@ -0,0 +1,49 @@
+set -x
+
+cd /home/vmagent/app/e2eaiok/e2eAIOK/deltatuner
+pip install -e .
+
+cd /home/vmagent/app/e2eaiok
+
+DATA_PATH="/home/vmagent/app/data"
+LOG_PATH=$DATA_PATH"/dtuner_test/log"
+MODEL_SAVE_PATH=$DATA_PATH"/dtuner_test/models"
+
+mkdir -p $LOG_PATH $MODEL_SAVE_PATH
+
+# fine-tune with ssf
+# gpt2 EleutherAI/gpt-j-6b bigscience/bloom-560m facebook/opt-125m EleutherAI/gpt-neo-125m tiiuae/falcon-7b
+# EleutherAI/gpt-j-6b tiiuae/falcon-7b
+# mosaicml/mpt-7b
+model_name_list="tiiuae/falcon-7b"
```

Comment:
why only add falcon, how about other models

Reply:
They are in the comments, please see the line 15. Actually, it's a temporary testing scripts, anyway, I update the `model_name_list`, please see commit  [2476b24](https://github.com/intel/e2eAIOK/pull/375/commits/2476b2490202b8b7cd932543cb7ba4187dd2fada).